.github/workflows/codeql-analysis.yml

```diff
@@ -1,28 +1,12 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
-name: "CodeQL"
-
+name: 'CodeQL'
 on:
-  push:
-    branches: [ main ]
   pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [ main ]
+    types: [ opened, reopened, synchronize ]
   schedule:
-    - cron: '36 12 * * 2'
+    - cron: '0 2 * * 7'
```

Comment:
Is this run at 2 every 7 days?

Reply:
It's 2AM Sunday instead of 12:36 (lunch) on a Tuesday